qa/remote-clusters/src/test/java/org/opensearch/cluster/remote/test/RemoteClustersIT.java

```diff
@@ -122,6 +123,9 @@ public void testHAProxyModeConnectionWorks() throws IOException {
 
         RemoteConnectionInfo rci = cluster1Client().cluster().remoteInfo(new RemoteInfoRequest(), RequestOptions.DEFAULT).getInfos().get(0);
         logger.info("Connection info: {}", rci);
+        if (!rci.isConnected()) {    
```

Comment:
I would have always logged cluster health here without the `if`, because it's a test, and you want to follow the same code path every time. But won't hold the PR for this, merge at will.